packages/manager/modules/pci/src/projects/project/storages/databases/fork/fork.controller.js

```diff
@@ -56,6 +58,10 @@ export default class {
       flavor: clone(flavor),
     };
 
+    this.backupSize = this.$filter('humanReadableSize')(
```

Comment:
is it possible to avoid $filter? maybe by using javascript/lodash filter? this will make our life little easy while migrating to Angular

Reply:
Done :)